DSCResources/SecurityPolicyResourceHelper/SecurityPolicyResourceHelper.psm1

```diff
@@ -320,3 +293,64 @@ function Test-IdentityIsNull
         return $false
     }
 }
+
+<#
+    .SYNOPSIS
+        Convert a SID to a common friendly name
+    .PARAMETER SID
+        SID of an identity being converted
+#>
+function ConvertTo-NTAccount
+{
+    [OutPutType([string])]
+    [CmdletBinding()] 
+    param
+    (
+        [parameter(mandatory=$true,ValueFromPipeline=$true)]
+        [System.Security.Principal.SecurityIdentifier[]]
+        $SID 
+    )
+
+    $result = @()
+    foreach ($id in $SID)
+    {
+        $id = ( $id -replace "\*" ).Trim()
+
+        $sidId = [System.Security.Principal.SecurityIdentifier]$id

```

Comment:
If the SID can't be converted the identity can't be applied so I was thinking this should be a terminating error.  

yes/no?